docs/_data/australia-2024-config.yaml

```diff
@@ -28,8 +28,8 @@ buttons:
   #     link: /news/announcing-virtual-conf-dates-tickets
   #   - text: Read our conference guide
   #     link: /welcome-wagon
-  #  - text: See the Schedule!
-  #    link: /schedule
```

Comment:
You'll want to comment out one of the other ones when you do this, so only two display.